### PR TITLE
Dispatch `MissingTranslationCategoryEvent` once per category

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by darkdef — APPROVED** [review]

**Comment by samdark** [comment]
@vjik would you please resolve conflicts?

**Comment by vjik** [comment]
> @vjik would you please resolve conflicts?

Done.

File: src/Translator.php [hunk 1]
```diff
@@ -27,6 +27,12 @@ final class Translator implements TranslatorInterface
 
     private ?SimpleMessageFormatter $simpleMessageFormatter = null;
 
+    /**
+     * @var array
```
Comment (arogachev) [inline src/Translator.php:31]: Is it needed? With the same type and no description. Rector should delete it.

Comment (vjik) [reply src/Translator.php]: Removed

**Review by arogachev — APPROVED** [review]

**Pull request merged**